cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -533,7 +536,19 @@ ClusterOperationMetadata createCluster(String name, String imageVersion, Map<Str
    */
   void updateClusterLabels(String clusterName,
                            Map<String, String> labels) throws RetryableProvisionException, InterruptedException {
-    if (labels.isEmpty()) {
+    updateClusterLabels(clusterName, labels, Collections.emptyList());
+  }
+  /**
```

Comment:
nit: add newline between previous method and this javadoc

Reply:
Done